Gemfile

```diff
@@ -1,3 +1,10 @@
 source 'https://rubygems.org'
 
+# for testing purposes
+if ENV['GEM_TESTING'] == '1'
+  git 'git@github.com:spree/spree.git', ref: 'b11bba9f616f' do
```

Comment:
That was master with all of the tests passing at the moment I was looking at it, can safely move to master to avoid confusion - I will submit another pull req.